# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/ash/src/extensions/ext/mod.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/ash/src/extensions/ext/mod.rs b/third_party/rust/ash/src/extensions/ext/mod.rs
--- a/third_party/rust/ash/src/extensions/ext/mod.rs
+++ b/third_party/rust/ash/src/extensions/ext/mod.rs
@@ -1,15 +1,23 @@
+pub use self::buffer_device_address::BufferDeviceAddress;
 #[allow(deprecated)]
 pub use self::debug_marker::DebugMarker;
 #[allow(deprecated)]
 pub use self::debug_report::DebugReport;
 pub use self::debug_utils::DebugUtils;
+pub use self::extended_dynamic_state::ExtendedDynamicState;
+pub use self::full_screen_exclusive::FullScreenExclusive;
 pub use self::metal_surface::MetalSurface;
+pub use self::physical_device_drm::PhysicalDeviceDrm;
 pub use self::tooling_info::ToolingInfo;
 
+mod buffer_device_address;
 #[deprecated(note = "Please use the [DebugUtils](struct.DebugUtils.html) extension instead.")]
 mod debug_marker;
 #[deprecated(note = "Please use the [DebugUtils](struct.DebugUtils.html) extension instead.")]
 mod debug_report;
 mod debug_utils;
+mod extended_dynamic_state;
+mod full_screen_exclusive;
 mod metal_surface;
+mod physical_device_drm;
 mod tooling_info;